website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -65,39 +73,37 @@ metrics:
-| filters     | A list of filters to apply before calculating the metric    | See below                       | no        |
-| meta        | Arbitrary key/value store                                   | {team: Finance}                 | no        |
-
-### Why define metrics?
-
-**Use metric specifications in downstream tools.** Metrics are available to dbt's compilation context via the [`graph.metrics` variable](graph). They are included in [the manifest artifact](manifest-json) for downstream metadata consumption.
+### Metrics properties
 
-**See and select dependencies.** As with exposures, it's possible to see everything that rolls up into a metric (`dbt ls -s +metric:*`), and visualize them in [dbt documentation](documentation).
+You can see how these properties are used in the [example YAML file](#declaring-a-metric).
 
-<Lightbox src="/img/docs/building-a-dbt-project/dag-metrics.png" title="Metrics appear as pink nodes in the DAG (for now)"/>
+| Field       | Description                                               | Example                           | Required? |
+|-------------|-----------------------------------------------------------|-----------------------------------|-----------|
+| name        | Unique identifier for the metric                          | new_customers                     | Yes       |
+| model       | The dbt model that powers this metric                     | dim_customers                     | Yes       |
+| label       | A short name / label for the metric                       | New Customers                     | No        |
+| description | Long form, human-readable description for the metric      | "The number of paid customers..." | No        |
+| type        | Type of calculation to perform when evaluating a metric   | count_distinct                    | Yes       |
+| sql         | Expression to aggregate or calculate over                 | user_id                           | Yes       |
+| timestamp   | Time-based component of the metric                        | signup_date                       | Yes       |
+| time_grains | One or more "grains" at which the metric can be evaluated | [day, week, month]                | Yes       |
+| dimensions  | List of dimensions to group or filter the metric by       | plan <br/> country                | No        |
+| filters     | List of filters to apply before calculating the metric    | field: is_paying<br/>value: true  | No        |
+| meta        | Arbitrary key/value store                                 | {team: Finance}                   | No        |
 
-### Open questions
+## Open questions
 
-- Should metrics be defined on top of more strongly typed **attributes**, rather than columns? [dbt-core#4090](https://github.com/dbt-labs/dbt-core/issues/4090)
+- Should we define metrics on top of more strongly typed **attributes**, rather than columns? [dbt-core#4090](https://github.com/dbt-labs/dbt-core/issues/4090)
```

Comment:
I linked to these issues to indicate that metrics are very much an open discussion, and to invite readers to contribute to that discussion. At the same time, I don't want to indicate that these are concrete next steps we plan on taking. I'd be curious to think together more about ways to mediate the conversation between docs ("this works this way") and issues ("this could work this other way")

Reply:
@jtcohen6 I _really_ like the spirit of this section! It brings the community back to the code.  

I wonder if framing this section as "Ongoing discussions" and linking to a GitHub Discussion in the `dbt-core` repo might be a good option so we don't have to hunt and pick through code comments to find decisions and context.  We could link to the discussion from the docs for people to see its status. I believe you can mark [discussions as "Answered"](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer) once it has been addressed or a decision made.

Reply:
Really neat idea. We haven't used `Discussions` to date (Drew and I played around with it way back when: https://github.com/dbt-labs/dbt-docs/discussions/156), but this feels like a good prompt to start using them. It's something I want to play around with more... probably after next week :)